apps/gateway-ui/src/components/WithdrawCard.tsx

```diff
@@ -96,35 +96,16 @@ export const WithdrawCard = React.memo(function WithdrawCard({
       });
   };
 
+  const description = `${t('withdraw-card.total_bitcoin')} ${
+    withdrawObject.amount / 100000
```

Comment:
Is this converting milli-sats to bits? Should we divide by 1000 to have just sats? OMG ARE YOU TEAM BITS?!?! 🤣

Reply:
I actually didn't write this code, I just pulled it out of the render into a var but it was originally written here: https://github.com/fedimint/ui/blob/dca2a26bb8817c075756e74354cd907ff4b46f23/apps/gateway-ui/src/components/WithdrawCard.tsx#L116. I haven't actually checked if the math is valid, but this should probably use a utility function instead of a one-off implementation due to how easy it is to be off by an order of magnitude or two.

Reply:
https://github.com/fedimint/ui/issues/182